src/graph/registerGraphCommands.ts

```diff
@@ -39,9 +39,15 @@ export function registerGraphCommands(context: vscode.ExtensionContext, tree: Az
         await node.deleteNode();
     });
     registerCommand('cosmosDB.openGraphExplorer', async (node: IAzureNode<GraphTreeItem>) => {
+        let treeItem: GraphTreeItem;
+        if (node) {
+            treeItem = node.treeItem;
+        }
         if (!node) {
-            node = <IAzureNode<GraphTreeItem>>await tree.showNodePicker(GraphCollectionTreeItem.contextValue);
+            let collectionNode = <IAzureNode<GraphCollectionTreeItem>>await tree.showNodePicker(GraphCollectionTreeItem.contextValue);
+            let children = await collectionNode.treeItem.loadMoreChildren(collectionNode, false);
+            treeItem = <GraphTreeItem>(children[0]);
```

Comment:
I believe the right way to do this is:

1) pass in GraphTreeItem to showNodePicker
2) Add it to GraphAccountTreeItem.isAncestorOf
3) Implement GraphCollectionTreeItem.pickTreeItem\

It's possible this might not work because of the fact that we don't want to show the final leaf node, but I think it will work.